src/test/java/org/recap/service/formatter/datadump/MarcXmlFormatterServiceUT.java

```diff
@@ -330,11 +320,7 @@ private BibliographicEntity getBibliographicEntity() throws URISyntaxException,
         bibliographicEntity.setLastUpdatedBy("tst");
         bibliographicEntity.setOwningInstitutionBibId("1");
         bibliographicEntity.setOwningInstitutionId(3);
-        InstitutionEntity institutionEntity = new InstitutionEntity();
-        institutionEntity.setId(1);
-        institutionEntity.setInstitutionCode("NYPL");
-        institutionEntity.setInstitutionName("New York Public Library");
-        bibliographicEntity.setInstitutionEntity(institutionEntity);
+        bibliographicEntity.setInstitutionEntity(getInstitutionEntity());
 
         HoldingsEntity holdingsEntity = new HoldingsEntity();
```

Comment:
Just holdingsEntity = new HoldingsEntity() should be good enough, since you already declared the variable at the top, and reutilize the same